src/watchkit.cs

```diff
@@ -913,4 +989,250 @@ interface WKAudioRecorderControllerOptionsKey {
+		void CrownDidRotate ([NullAllowed] WKCrownSequencer crownSequencer, double rotationalDelta);
+
+		[Export ("crownDidBecomeIdle:")]
+		void CrownDidBecomeIdle ([NullAllowed] WKCrownSequencer crownSequencer);
+	}
+
+	[Watch (3,0)][NoiOS]
+	[Abstract]
+	[BaseType (typeof (NSObject))]
+	interface WKGestureRecognizer {
+
+		[Export ("state")]
+		WKGestureRecognizerState State { get; }
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Export ("locationInObject")]
+		CGPoint LocationInObject { get; }
+
+		[Export ("objectBounds")]
+		CGRect ObjectBounds { get; }
+	}
+
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKGestureRecognizer))]
+	interface WKTapGestureRecognizer {
+
+		[Export ("numberOfTapsRequired")]
+		nuint NumberOfTapsRequired { get; set; }
+	}
+
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKGestureRecognizer))]
+	interface WKLongPressGestureRecognizer {
+
+		[Export ("minimumPressDuration")]
+		double MinimumPressDuration { get; set; }
+
+		[Export ("numberOfTapsRequired")]
+		nuint NumberOfTapsRequired { get; set; }
+
+		[Export ("allowableMovement")]
+		nfloat AllowableMovement { get; set; }
+	}
+
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKGestureRecognizer))]
+	interface WKSwipeGestureRecognizer {
+
+		[Export ("direction", ArgumentSemantic.Assign)]
+		WKSwipeGestureRecognizerDirection Direction { get; set; }
+	}
+
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKGestureRecognizer))]
+	interface WKPanGestureRecognizer {
+		[Export ("translationInObject")]
+		CGPoint TranslationInObject { get; }
+
+		[Export ("velocityInObject")]
+		CGPoint VelocityInObject { get; }
+	}
+
+#if false // FIXME HomeKit not yet enabled on platform
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKInterfaceObject))]
+	interface WKInterfaceHMCamera {
+
+		[Export ("setCameraSource:")]
+		void SetCameraSource ([NullAllowed] HMCameraSource cameraSource);
+	}
+#endif
+
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKInterfaceObject))]
+	interface WKInterfaceInlineMovie {
+
+		[Export ("setMovieURL:")]
+		void SetMovieUrl (NSUrl url);
+
+		[Export ("setVideoGravity:")]
+		void SetVideoGravity (WKVideoGravity videoGravity);
+
+		[Export ("setLoops:")]
+		void SetLoops (bool loops);
+
+		[Export ("setAutoplays:")]
+		void SetAutoplays (bool autoplays);
+
+		[Export ("setPosterImage:")]
+		void SetPosterImage ([NullAllowed] WKImage posterImage);
+
+		[Export ("play")]
+		void Play ();
+
+		[Export ("playFromBeginning")]
+		void PlayFromBeginning ();
+
+		[Export ("pause")]
+		void Pause ();
+	}
+
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKInterfaceObject))]
+	interface WKInterfacePaymentButton {
+	}
+
+#if false // FIXME SceneKit not yet enabled on platform
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKInterfaceObject))]
+	interface WKInterfaceSCNScene : ISCNSceneRenderer {
+
+		[NullAllowed, Export ("scene", ArgumentSemantic.Retain)]
+		SCNScene Scene { get; set; }
+
+		UIImage GetSnapshot ();
```

Comment:
I know this is inside an `false` but maybe adding a Export or a comment here so we do not miss on a future review